ksql-common/src/main/java/io/confluent/ksql/util/DecimalUtil.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import org.apache.kafka.connect.data.Decimal;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Schema.Type;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.errors.DataException;
+
+public final class DecimalUtil {
+
+  private static final String PRECISION_FIELD = "precision";
```

Comment:
It would be better to add the "precision" in the Connect side instead. Avro uses both, precision and scale, parameters, but Connect decided to set a default to 'precision' and rename the parameter, but let scale as it is. I think that's confusing, and adding a new precision parameter name will add more confusion, don't you think?

Reply:
The plan is to add `precision"` to the connect class itself and the collision will be intentional (I'd then follow-up removing this constant from KSQL)

Reply:
After seeing how `"connect.decimal.precision"` is used, I think it makes sense to use that here in the connect schema as the `PRECISION_FIELD` - either way this is temporary until we change connect to have first class support for precision